pkgs/top-level/aliases.nix

```diff
@@ -1151,6 +1151,7 @@ mapAliases ({
   tex-gyre-termes-math = throw "'tex-gyre-termes-math' has been renamed to/replaced by 'tex-gyre-math.termes'"; # Converted to throw 2022-02-22
   textadept11 = textadept; # Added 2022-06-07
   tftp_hpa = throw "'tftp_hpa' has been renamed to/replaced by 'tftp-hpa'"; # Converted to throw 2022-02-22
+  timedoctor = throw "'timedoctor' has been removed from nixpkgs"; # Added 2021-06-20
```

Comment:
Is this the date that this package was added to nixpkgs? I think the date in there is the date where the package is added to aliases.

Reply:
Ah, TIL. Changed to today's date.